.github/workflows/publish-crate.yml

```diff
@@ -0,0 +1,20 @@
+name: Continuous Deployment
+
+on:
+  push:
+    tags:
+      - "v*.*.*"
+
+jobs:
+  publish-stable:
+    name: Create a stable release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v4
+
+      - name: Publish on crates.io
+        uses: actions-rs/cargo@v1
```

Comment:
This action is deprecated / archived since 2023: https://github.com/actions-rs/cargo. Instead it's probably a good idea to install rust using https://github.com/dtolnay/rust-toolchain and then calling cargo publish from a run step.